smaug/__init__.py

```diff
@@ -9,3 +15,7 @@
 from smaug.python.graph import Graph
 from smaug.python.tensor import Tensor
 from smaug.python.node import Node
+
+def _autodoc_for_module():
```

Comment:
I remember I couldn't get this working and used `autoclass` and `autofunction` instead.

Reply:
If I wanted this function's documentation to appear, I just need to add `:members:` under the `automodule` section in smaug.rst. But I don't - I just need something under this namespace to exist so that Sphinx will generate a link for the entire module.